docs/root/source/transaction-concepts.md

```diff
@@ -4,31 +4,47 @@ In BigchainDB, _Transactions_ are used to register, issue, create or transfer
 things (e.g. assets).
 
 Transactions are the most basic kind of record stored by BigchainDB. There are
-two kinds: creation transactions and transfer transactions.
+two kinds: CREATE transactions and TRANSFER transactions.
 
-A _creation transaction_ can be used to register, issue, create or otherwise
+## CREATE Transactions
+
+A CREATE transaction can be used to register, issue, create or otherwise
 initiate the history of a single thing (or asset) in BigchainDB. For example,
 one might register an identity or a creative work. The things are often called
 "assets" but they might not be literal assets.
 
-Currently, BigchainDB only supports indivisible assets. You can't split an
-asset apart into multiple assets, nor can you combine several assets together
-into one. [Issue #129](https://github.com/bigchaindb/bigchaindb/issues/129) is
-an enhancement proposal to support divisible assets.
+BigchainDB supports divisible assets as of BigchainDB Server v0.8.0.
+That means you can create/register an asset with an initial quantity,
+e.g. 700 thumbtacks. Divisible assets can be split apart or recombined
+by transfer transactions (described more below).
 
-A creation transaction also establishes the conditions that must be met to
-transfer the asset. For example, there may be a condition that any transfer
+A CREATE transaction also establishes the conditions that must be met to
+transfer the asset(s). For example, there may be a condition that any transfer
 must be signed (cryptographically) by the private key associated with a
 given public key. More sophisticated conditions are possible.
 BigchainDB's conditions are based on the crypto-conditions of the [Interledger
 Protocol (ILP)](https://interledger.org/).
 
-A _transfer transaction_ can transfer an asset by fulfilling the current
-conditions on the asset. It can also specify new transfer conditions.
+## TRANSFER Transactions
+
+A TRANSFER transaction can transfer an asset
+(or set of assets of the same type)
```

Comment:
I find this sentence confusing because at this point a transaction is related to a particular asset.
I understand that what you are trying to say is that I can spend multiple outputs related to the same divisible asset but I wonder if there is a better language for this